examples/cms-sanity/README.md

```diff
@@ -0,0 +1,140 @@
+# A statically generated blog example using Next.js and Sanity
+
+This example showcases Next.js's [Static Generation](/docs/basic-features/pages.md) feature using [Sanity](https://www.sanity.io/) as the data source.
+
+## Demo
+
+### [https://next-blog-sanity.now.sh/](https://next-blog-sanity.now.sh/)
+
+## How to use
+
+### Using `create-next-app`
+
+Execute [`create-next-app`](https://github.com/zeit/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npm init next-app --example cms-Sanity cms-Sanity-app
+# or
+yarn create next-app --example cms-Sanity cms-Sanity-app
+```
+
+### Download manually
+
+Download the example:
+
+```bash
+curl https://codeload.github.com/zeit/next.js/tar.gz/canary | tar -xz --strip=2 next.js-canary/examples/cms-Sanity
+cd cms-sanity
+```
+
+## Configuration
+
+### Step 1. Create an account and a project on Sanity
+
+First, [create an account on Sanity](https://sanity.io).
+
+After creating an account, install the Sanity cli from npm `npm i -g @sanity/cli`.
+
+### Step 2. Create a new Sanity project
+
+In a separate folder run `sanity init` to initialize a new studio project.
+
+This will be where we manage our data.
+
+When going through the init phase make sure to select **Yes** to the **Use the default dataset configuration** step and select **Clean project with no predefined schemas** for the **Select project template** step.
+
+After initializing get the `projectId` and `dataset` values from the `sanity.json` file that was generated and add them to `lib/sanity.js` in the example.
+
+### Step 3. Copy the schema file
+
+After initializing your Sanity studio project there should be a `schemas` folder.
+
+To get up and running we want to copy the schema file from [`./schemas/schema.js`](./schemas/schema.js) to the studio project to use for creating/editing content.
+
+### Step 4. Populate Content
+
+To add some content go to your Sanity studio project where we added the schema file and run `sanity start`.
+
+After the project has started and you have navigated to the URL given in the terminal, select **Author** and create a new record.
+
+- You just need **1 Author record**.
+- Use dummy data for the text.
+- For the image, you can download one from [Unsplash](https://unsplash.com/).
+
+Next, select **Post** and create a new record.
+
+- We recommend creating at least **2 Post records**.
+- Use dummy data for the text.
+- You can write markdown for the **Content** field.
+- For the images, you can download ones from [Unsplash](https://unsplash.com/).
+- Pick the **Author** you created earlier.
+
+**Important:** For each post record, you need to click **Publish** after saving. If not, the post will be in the draft state.
```

Comment:
> **Important:** For each post record, you need to click **Publish** after saving. If not, the post will be in the draft state.

This was true for Dato, but is this true for Sanity?

Reply:
Sanity have the same behaviour yes